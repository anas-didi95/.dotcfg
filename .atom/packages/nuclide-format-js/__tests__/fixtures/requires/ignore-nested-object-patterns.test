function foo({one: {two, three: [four]}}) {
  five(two, four);
}
