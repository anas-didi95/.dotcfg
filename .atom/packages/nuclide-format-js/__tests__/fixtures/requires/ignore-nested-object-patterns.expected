const five = require('five');

function foo({one: {two, three: [four]}}) {
  five(two, four);
}
